CMakeLists.txt

```diff
@@ -380,6 +380,8 @@ set(CPACK_IGNORE_FILES "\\\\.a;/\\\\.ccache/;/\\\\.git/;${RTTR_OUTPUT_DIR};${PRO
 set(CPACK_SOURCE_IGNORE_FILES "${CPACK_IGNORE_FILES}")
 set(CPACK_BUILD_SOURCE_DIRS "${PROJECT_SOURCE_DIR}")
 
+set(CPACK_STRIP_FILES ON)
```

Comment:
shouldn't it only do that if not explicitly build as debug?

Reply:
It will only strip the files when running `make install/strip` or when using cpack via `make package`. IMO this is ok.

Making it only do that in Release mode will not work with multi-generator targets.

Reply:
ah okay then its okay